encord/objects/ontology_structure.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from enum import IntEnum
+from typing import List, TypeVar
+from uuid import uuid4
+
+from encord.objects.classification import Classification
+from encord.objects.common import Shape
+from encord.objects.ontology_object import Object
+
+AVAILABLE_COLORS = (
+    "#D33115",
+    "#E27300",
+    "#16406C",
+    "#FE9200",
+    "#FCDC00",
+    "#DBDF00",
+    "#A4DD00",
+    "#68CCCA",
+    "#73D8FF",
+    "#AEA1FF",
+    "#FCC400",
+    "#B0BC00",
+    "#68BC00",
+    "#16A5A5",
+    "#009CE0",
+    "#7B64FF",
+    "#FA28FF",
+    "#B3B3B3",
+    "#9F0500",
+    "#C45100",
+    "#FB9E00",
+    "#808900",
+    "#194D33",
+    "#0C797D",
+    "#0062B1",
+    "#653294",
+    "#AB149E",
+)
+
+
+@dataclass
```

Comment:
I'm not convinced whether this should be called `Ontology` and the new class something else. But I'll leave it to you and @rad-cord to find the best name

Reply:
The reasoning here is to match the naming in the UX: "Ontology" is the thing that has title and links to user and an `ontology_hash`, etc, and then we call the tab where we show actual objects the "Structure".

If we use the name "Ontology" on what used to be `editor` then this doesn't match UX or DB. We could flatten this all up and just have `objects` and `classifications` properties on the same level as `title`, etc, but then we'd have issues with serialisation and `from_dict`. It would be super silly to have the JSON display in the UX and then not to be able to use that same JSON with the SDK.

Reply:
Agreed on the JSON in the UI and the SDK matching.